### PR TITLE
Adding a license file.

### PR DESCRIPTION
**Pull request by jkamenik**
I have no opinion of what license you should be using, but I do think that each project (especially useful ones) should have a license.  I choose MIT as the default for my own stuff, but if you prefer a different license I would be happy to change it.

I updated the Copyright line to include your name (taken from your profile) and the year of the first commit (taken from `git log`).

Let me know if you want anything changed.

### CODE REVIEW

**Comment by Vinzent03** [comment]
Hmm, I'm maintaining this plugin for the last two years now, but I don't know if I'm theoretically allowed to decide that for him. So I guess I have to ask @denolehov. Is MIT okay for you?

**Comment by denolehov** [comment]
MIT is okay 👍

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Denis Olehov
```
Comment (denolehov) [inline LICENSE:3]: @Vinzent03 we must add your name here as well, and it should be before mine :D

Comment (denolehov) [reply LICENSE:3]: Given that your handle is technically PII, it is possible to use it (according to my quick google search :D)

**Comment by jkamenik** [comment]
I have updated to include both of you in the order requested.  Thank you.

**Comment by Vinzent03** [comment]
LGTM! thanks

**Comment by denolehov** [comment]
Thank you @jkamenik! 🙌

**Pull request merged**